superset/views/core.py

```diff
@@ -1421,6 +1421,48 @@ def fave_slices(  # pylint: disable=no-self-use
             payload.append(dash)
         return json_success(json.dumps(payload, default=utils.json_int_dttm_ser))
 
+    @api
+    @has_access_api
+    @event_logger.log_this
+    @expose("/dashboard/<dashboard_id>/stop/", methods=["POST"])
+    def stop_dashboard_queries(  # pylint: disable=no-self-use
```

Comment:
It may be useful to sync with @dpgaspar on using api v1. 
Also please add some tests, we have presto on CI

Reply:
note, in open source code base, presto's `stop_queries` function is empty, nothing to test. airbnb and other companies can add their internal implementation to stop queries by dashboard_id, but this is not a standard Presto API.

Reply:
it would be nice to have stop_queries implementation in the test config.
It will help in 2 ways:
1. be a safeguard from open source contributions not to break it
2. will work as an example for the companies willing to try it out

Reply:
do you have a exist example, what is `implementation in the test config`? thanks!

Reply:
yep, custom templates:
https://github.com/apache/incubator-superset/blob/master/tests/superset_test_config.py#L100
cta schema:
https://github.com/apache/incubator-superset/blob/master/tests/celery_tests.py#L176

Reply:
I think as I mentioned above, core apis are deprecated and new ones should be added here: https://github.com/apache/incubator-superset/blob/master/superset/dashboards/api.py @dpgaspar & @villebro would probably know more on this topic